src/app/pages/admin/twins/details/twins.details.component.html

```diff
@@ -55,85 +83,134 @@
-              <li class="row"><b class="col-md-2">Persisted</b>
-                <span class="col-md-5">{{ attr.persist_state }}</span></li>
-            </ul>
+        <nb-list-item>
+          <div class="col-3">
+            <b>Name</b>
+          </div>
+          <div class="col-4">
+            <b>Channel</b>
+          </div>
+          <div class="col-3">
+            <b>Subtopic</b>
+          </div>
+          <div class="col-1">
+            <b>Persisted</b>
+          </div>
+          <div class="col-1">
+          </div>
+        </nb-list-item>
+        <nb-list-item *ngFor="let attr of editAttrs">
+          <div class="col-3">
+            {{ attr.name }}
+          </div>
+          <div class="col-4">
+            {{ attr.channel }}
+          </div>
+          <div class="col-3">
+            {{ attr.subtopic }}
+          </div>
+          <div class="col-1">
+            <b>{{ attr.persist_state }}</b>
           </div>
           <div class="col-1">
             <button (click)="removeAttribute(attr)" nbButton status="danger" size="tiny">x</button>
           </div>
-        </div>
+        </nb-list-item>
       </nb-card-body>
 
       <nb-card-footer>
         <div class="row">
-          <div class="col-2">Name: </div><div class="col-10">
+          <div class="col-1">
+            <b>Name:</b>
+          </div>
+          <div class="col-3">
             <input type="text" nbInput [(ngModel)]="editAttr.name" fieldSize="small">
           </div>
 
-          <div class="col-2">Channel: </div><div class="col-10">
-            <nb-select placeholder="Select channel" [(selected)]="editAttr.channel" size="tiny">
+          <div class="col-1">
+            <b>Channel:</b>
+          </div>
+          <div class="col-6">
+            <nb-select placeholder="Select channel" [(selected)]="editAttr.channel" size="small">
               <nb-option *ngFor="let channel of this.channels" [value]="channel.id">
                 {{channel.name}}
               </nb-option>
             </nb-select>
           </div>
+        </div>
 
-          <div class="col-2">Subtopic: </div><div class="col-10">
+        <div class="row">
+          <div class="col-1">
+            <b>Subtopic:</b>
+          </div>
+          <div class="col-3">
             <input type="text" nbInput [(ngModel)]="editAttr.subtopic" fieldSize="small">
           </div>
-
-          <div class="col-2">Persisted: </div><div class="col-5">
+          <div class="col-1">
+            <b>Persisted:</b>
+          </div>
+          <div class="col-2">
             <nb-checkbox (checkedChange)="togglePersist($event)" [checked]="editAttr.persist_state"></nb-checkbox>
           </div>
         </div>
 
-        <button (click)="addAttribute()" nbButton status="success" size="tiny">
-          Add/Update attribute
+        <button (click)="addAttribute()" nbButton size="small">
+          Add / Update attribute
```

Comment:
somehow I prefer Add/Update (without blanks)

Reply:
it's not readable in the button...

Reply:
Ok, if that is the case, then it is ok for me